src/main/java/eu/dissco/backend/controller/MasJobRecordController.java

```diff
@@ -0,0 +1,56 @@
+package eu.dissco.backend.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.backend.domain.AnnotationState;
+import eu.dissco.backend.domain.jsonapi.JsonApiListResponseWrapper;
+import eu.dissco.backend.domain.jsonapi.JsonApiWrapper;
+import eu.dissco.backend.exceptions.NotFoundException;
+import eu.dissco.backend.properties.ApplicationProperties;
+import eu.dissco.backend.service.MasJobRecordService;
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@Slf4j
+@RestControllerAdvice
+@RestController
+@RequestMapping("api/v1/mjr")
+public class MasJobRecordController extends BaseController {
+
+  private final MasJobRecordService service;
+
+  public MasJobRecordController(ObjectMapper mapper,
+      ApplicationProperties applicationProperties, MasJobRecordService service) {
+    super(mapper, applicationProperties);
+    this.service = service;
+  }
+
+  @GetMapping(value = "/{jobId}", produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<JsonApiWrapper> getMasJobRecord(
+      @PathVariable("jobId") UUID jobId, HttpServletRequest request) throws NotFoundException {
+    return ResponseEntity.ok().body(service.getMasJobRecordById(jobId, getPath(request)));
+  }
+
+  @GetMapping(value = "/creator/"
```

Comment:
Wonder if this is a call/operation on the user object and should therefore be in the user controller. What do you think?

Reply:
"Creator" here is the MAS Id, not user that called the user that scheduled the MAS, so I think this endpoint is the best. 
We don't actually capture the user information in the Mas Job Record. I can a user might want to see all their scheduled jobs. Added a [new jira story](https://naturalis.atlassian.net/browse/DD-835) to modify this object

Reply:
Oh, yes sorry that would be nice, let's add that info :+1: